src/Futura_frontend/src/App.jsx

```diff
@@ -1,5 +1,5 @@
 import { useState, useEffect } from "react";
-import { futura_backend } from "declarations/futura_backend";
+import { Futura_backend } from "declarations/futura_backend";
```

Comment:
The import are correct like these. Originally the package was called Futura_backend, cause @lmangall created the new project with `dfx new Futura` and not with `dfx new futura`, low-case. When I deployed the code, I got a warning that packages should be name snake_case. So I renamed everything what it should be renamed. But then you need to generate I guess a new identity or something like that for the backend canister. So `futura_backend` instead of `Futura_backened` is not an error, and it should be like this. Best options would be rename also the folder accordingly or restart it from scratch with `dfx new futura`.

Reply:
Code does not work with Futura_backend - I get errors

Reply:
Which errors?

Reply:
Cause they are not commited.

Reply:
Anyway you have to do more than just dfx build or dfx deploy